pkg/cnab/provider/parameters.go

```diff
@@ -102,34 +107,46 @@ func (r *Runtime) resolveParameterSources(bun bundle.Bundle, args ActionArgument
 	values := valuesource.Set{}
 	for parameterName, parameterSource := range parameterSources {
 		for _, rawSource := range parameterSource.ListSourcesByPriority() {
+			var installation string
+			var outputName string
 			switch source := rawSource.(type) {
 			case extensions.OutputParameterSource:
-				output, err := r.claims.ReadLastOutput(args.Installation, source.OutputName)
-				if err != nil {
-					// When we can't find the output, skip it and let the parameter be set another way
-					if strings.Contains(err.Error(), claim.ErrInstallationNotFound.Error()) ||
-						strings.Contains(err.Error(), claim.ErrOutputNotFound.Error()) {
-						continue
-					}
-					// Otherwise, something else has happened, perhaps bad data or connectivity problems, we can't ignore it
-					return nil, errors.Wrapf(err, "could not set parameter %s from its parameter source, output %s", parameterName, source.OutputName)
-				}
+				installation = args.Installation
+				outputName = source.OutputName
+			case extensions.DependencyOutputParameterSource:
+				installation = fmt.Sprintf("%s-%s", args.Installation, source.Dependency)
```

Comment:
Do we have a helper yet around this dep naming convention?  We do same/similar elsewhere, right?

Reply:
No we don't have a helper and I totally messed up that naming pattern once already when developing this locally. Let me add a helper function for this.